src/components/NavBar/NavBarTools.tsx

```diff
@@ -1,18 +1,19 @@
 import { Link } from 'react-router-dom';
 import { tools } from '../../assets/pages';
+import DropDownStyle from '../BasicComponents/DropDownMenu.module.css';
 
 interface ToolsProps {
   platform: string;
 }
 
 const NavBarTools = ({ platform }: ToolsProps) => (
-  <div className="dropdown-menu">
-    <div className="dropdown-menu-item">
+  <div className={DropDownStyle.dropdownMenu}>
+    <div className={DropDownStyle.dropdownMenuItem}>
```

Comment:
dose the items( the links in this scenario) not supposed to have the **item** class , and not what warp  them?